code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -47,3 +47,44 @@
 			else
 				to_chat(user, SPAN_NOTICE("Nothing to fix here."))
 
+/obj/item/stack/nanopaste/weak
+	name = "low power nanopaste"
+	singular_name = "low power nanite swarm"
+	desc = "A cheap alternative to nanopaste. Fairly effective in repairing robotic machinery."
+
+	black_market_value = 10
+
+/obj/item/stack/nanopaste/weak/attack(mob/living/Mob as mob, mob/user as mob)
```

Comment:
dont copy paste entire procs to change some number - make var/heal_amount on the parent type and adjust it on the subtype, and use heal_amount in the parent attack proc

Reply:
also this isn’t snakecase